src/runtime-rs/crates/runtimes/virt_container/src/container_manager/container_inner.rs

```diff
@@ -157,7 +159,36 @@ impl ContainerInner {
         }
     }
 
-    async fn cleanup_container(&mut self, cid: &str, force: bool) -> Result<()> {
+    async fn remove_device(
+        &mut self,
+        _force: bool,
```

Comment:
Should ignore some errors by log a warning message if `force` is set to true when some operation failed?